doc/075_scripting.rst

```diff
@@ -214,9 +214,9 @@ Summary is the last output line in a successful backup.
 +---------------------------+---------------------------------------------------------+
 | ``dirs_unmodified``       | Number of directories that did not change               |
 +---------------------------+---------------------------------------------------------+
-| ``data_blobs``            | Number of data blobs                                    |
+| ``data_blobs``            | Number of data blobs added by this snapshot             |
```

Comment:
Hm, I think "by this snapshot" is unnecessary. The context is already quite given, in particular when the text is changed to contain "added" at the end - there can't be much misunderstanding that the adding must have been done in the newly created snapshot. Also, the other descriptions are of the same type, they talk about what has changed/been added. I therefore suggest:

```suggestion
| ``data_blobs``            | Number of data blobs added                              |
```

Reply:
I thought about that and chose the more explicit version. But I agree, it's more in line with the other descriptions and from the context it's clear.

Reply:
But the other occurrence should be identical.

Reply:
Hm, what do you mean by that the other occurrence should be identical? Shouldn't all of the changes you made be adjusted to end with "added"?

Reply:
Yes. At first I saw only one suggested change and accepting it changed only this line. Now I added a commit which changes the other 3. Feel free to squash it into a single commit.